protocol/x/clob/abci.go

```diff
@@ -104,6 +104,12 @@ func EndBlocker(
 
 	// Prune any rate limiting information that is no longer relevant.
 	keeper.PruneRateLimits(ctx)
+
+	telemetry.ModuleSetGauge(
```

Comment:
Question: do you think this `ModuleSetGauge` function is helpful? IIRC most of our metrics are pre-pended with the module name, and this function [adds the module name as a label](https://github.com/dydxprotocol/cosmos-sdk/blob/93454d9f2fb6f262e63a55641e44c024bd326860/telemetry/wrapper.go#L36) which IMO makes it harder to find module metrics since we mix styles

I prefer [`SetGaugeWithLabels`](https://github.com/dydxprotocol/cosmos-sdk/blob/93454d9f2fb6f262e63a55641e44c024bd326860/telemetry/wrapper.go#L64) for this reason

Reply:
no preference here. updated.